### PR TITLE
Update changelog.md for 14.0

### PR DESCRIPTION
**Pull request by rchande**

### CODE REVIEW

**Comment by rchande** [comment]
cc @akshita31 @TheRealPiotrP

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,10 +14,22 @@
 #### Editor
 
 * Fix to allow signature help return results for attribute constructors. ([#1814](https://github.com/OmniSharp/omnisharp-vscode/issues/1814), PR: [omnisharp-roslyn#1007](https://github.com/OmniSharp/omnisharp-roslyn/pull/1007))
+* Fix to return correct SymbolKinds from WorkspaceSymbolprovider ([#1907](https://github.com/OmniSharp/omnisharp-vscode/issues/1907), PR: [#1911](https://github.com/OmniSharp/omnisharp-vscode/pull/1911)) (_Contributed by [@craig006](https://github.com/craig006)_)
+* Improved newline formatting in hover info ([#1057](https://github.com/OmniSharp/omnisharp-vscode/issues/1057), PR:[1918](https://github.com/OmniSharp/omnisharp-vscode/pull/1918))
+
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Extra blank line

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,10 +14,22 @@
...
+* Added option to disable warning about project.json deprecation ([1920](https://github.com/OmniSharp/omnisharp-vscode/issues/1920), PR: [1926](https://github.com/OmniSharp/omnisharp-vscode/pull/1926))
+
+#### Task Generation
+* Updated task generator to match latest schema from VS Code (PR: [#1932](https://github.com/OmniSharp/omnisharp-vscode/pull/1923)) (_Contributed by [@natec425](https://github.com/natec425)_)
+* Fixed a typo in tasks.json (PR: [1945](https://github.com/OmniSharp/omnisharp-vscode/pull/1945)) (_Contributed by [@SebastianPfliegel](SebastianPfliegel)_)
+
+#### Misc
+* Fixed offline packaging ([1912](https://github.com/OmniSharp/omnisharp-vscode/issues/1912), [1930](https://github.com/OmniSharp/omnisharp-vscode/issues/1930), PR:]1931](https://github.com/OmniSharp/omnisharp-vscode/pull/1931))
+
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Extra blank line

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,10 +14,22 @@
...
+* Improved newline formatting in hover info ([#1057](https://github.com/OmniSharp/omnisharp-vscode/issues/1057), PR:[1918](https://github.com/OmniSharp/omnisharp-vscode/pull/1918))
+
 
 #### Project System
 
 * Addressed problems with projects not being refreshed by OmniSharp after a package restore. ([#1583](https://github.com/OmniSharp/omnisharp-vscode/issues/1583), [#1661](https://github.com/OmniSharp/omnisharp-vscode/issues/1661), [#1785](https://github.com/OmniSharp/omnisharp-vscode/issues/1785), PR: [omnisharp-roslyn#1003](https://github.com/OmniSharp/omnisharp-roslyn/pull/1003))
+* Added option to disable warning about project.json deprecation ([1920](https://github.com/OmniSharp/omnisharp-vscode/issues/1920), PR: [1926](https://github.com/OmniSharp/omnisharp-vscode/pull/1926))
+
+#### Task Generation
+* Updated task generator to match latest schema from VS Code (PR: [#1932](https://github.com/OmniSharp/omnisharp-vscode/pull/1923)) (_Contributed by [@natec425](https://github.com/natec425)_)
```
Comment (DustinCampbell) [inline CHANGELOG.md]: minor nit: Should the italics go _outside_ the parentheses? It looks like that's how there were done before.

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,10 +14,22 @@
...
+* Fix to return correct SymbolKinds from WorkspaceSymbolprovider ([#1907](https://github.com/OmniSharp/omnisharp-vscode/issues/1907), PR: [#1911](https://github.com/OmniSharp/omnisharp-vscode/pull/1911)) (_Contributed by [@craig006](https://github.com/craig006)_)
+* Improved newline formatting in hover info ([#1057](https://github.com/OmniSharp/omnisharp-vscode/issues/1057), PR:[1918](https://github.com/OmniSharp/omnisharp-vscode/pull/1918))
+
 
 #### Project System
 
 * Addressed problems with projects not being refreshed by OmniSharp after a package restore. ([#1583](https://github.com/OmniSharp/omnisharp-vscode/issues/1583), [#1661](https://github.com/OmniSharp/omnisharp-vscode/issues/1661), [#1785](https://github.com/OmniSharp/omnisharp-vscode/issues/1785), PR: [omnisharp-roslyn#1003](https://github.com/OmniSharp/omnisharp-roslyn/pull/1003))
+* Added option to disable warning about project.json deprecation ([1920](https://github.com/OmniSharp/omnisharp-vscode/issues/1920), PR: [1926](https://github.com/OmniSharp/omnisharp-vscode/pull/1926))
+
+#### Task Generation
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Missing blank line

File: CHANGELOG.md [hunk 5]
```diff
@@ -14,10 +14,22 @@
...
+* Updated task generator to match latest schema from VS Code (PR: [#1932](https://github.com/OmniSharp/omnisharp-vscode/pull/1923)) (_Contributed by [@natec425](https://github.com/natec425)_)
+* Fixed a typo in tasks.json (PR: [1945](https://github.com/OmniSharp/omnisharp-vscode/pull/1945)) (_Contributed by [@SebastianPfliegel](SebastianPfliegel)_)
+
+#### Misc
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Missing blank line

File: CHANGELOG.md [hunk 6]
```diff
@@ -14,10 +14,22 @@
 #### Editor
 
 * Fix to allow signature help return results for attribute constructors. ([#1814](https://github.com/OmniSharp/omnisharp-vscode/issues/1814), PR: [omnisharp-roslyn#1007](https://github.com/OmniSharp/omnisharp-roslyn/pull/1007))
+* Fix to return correct SymbolKinds from WorkspaceSymbolprovider ([#1907](https://github.com/OmniSharp/omnisharp-vscode/issues/1907), PR: [#1911](https://github.com/OmniSharp/omnisharp-vscode/pull/1911)) (_Contributed by [@craig006](https://github.com/craig006)_)
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Same question about italics here.

File: CHANGELOG.md [hunk 7]
```diff
@@ -14,10 +14,22 @@
...
+
 
 #### Project System
 
 * Addressed problems with projects not being refreshed by OmniSharp after a package restore. ([#1583](https://github.com/OmniSharp/omnisharp-vscode/issues/1583), [#1661](https://github.com/OmniSharp/omnisharp-vscode/issues/1661), [#1785](https://github.com/OmniSharp/omnisharp-vscode/issues/1785), PR: [omnisharp-roslyn#1003](https://github.com/OmniSharp/omnisharp-roslyn/pull/1003))
+* Added option to disable warning about project.json deprecation ([1920](https://github.com/OmniSharp/omnisharp-vscode/issues/1920), PR: [1926](https://github.com/OmniSharp/omnisharp-vscode/pull/1926))
+
+#### Task Generation
+* Updated task generator to match latest schema from VS Code (PR: [#1932](https://github.com/OmniSharp/omnisharp-vscode/pull/1923)) (_Contributed by [@natec425](https://github.com/natec425)_)
+* Fixed a typo in tasks.json (PR: [1945](https://github.com/OmniSharp/omnisharp-vscode/pull/1945)) (_Contributed by [@SebastianPfliegel](SebastianPfliegel)_)
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Same question about italics here.

**Comment by filipw** [comment]
> Looks pretty good. Are there fixes from omnisharp-roslyn that we should be mentioning and linking here?

I think C# 7.2 support should be mentioned because people are asking about it. Also, perhaps code action ordering fixes because it has direct impact on the UX.

**Comment by DustinCampbell** [comment]
> I think C# 7.2 support should be mentioned because people are asking about it. Also, perhaps code action ordering fixes because it has direct impact on the UX.

Agreed.

**Comment by akshita31** [comment]
@rchande Can we close this ?

**Pull request closed without merging**